### PR TITLE
Gui/graphcontroller

### PR DESCRIPTION
**Pull request by HugoKooijman**
The graph now appears to be bug-free. All of the graph is still loaded at once onto the graph tab in the graphical user interface, so unfortunately there is significant fps lag when scrolling over the graph. We will improve this with Semantic zooming, when we will only load parts of the graph on screen.

### CODE REVIEW

**Comment by HatchIing** [comment]
Looks like it works well now. Glad that you and Bjorn were able to fix the scaling. There are checkstyle errors but those should be fixed for the most part already in a different branch. If there are any warnings I'll fix them after the merge. So it looks like it's ready to me.

**Comment by RobKapel** [comment]
It seems that you are not able to zoom out, so that you can see more segments at once. Is this intended?

**Comment by HugoKooijman** [comment]
@RobKapel Yes, this functionality is handled by @DvdBerg, so I did not really tweak the ScrollPane. I simply made a very basic one to see whether the graph works properly. This should be fixed, once we merge @DvdBerg's branch #48 with mine.

**Comment by RobKapel** [comment]
Okay, thanks for the clarification, then everything seems fine to me!

File: src/main/java/gui/GraphSegment.java [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+	public void setDnacontent(char[] dnacontent) {
+		this.dnacontent = dnacontent;
+	}
+	
+	public double getRadius() {
+		return this.image.getRadius();
+	}
+	
+//	
+//	public GraphSeg() {
```
Comment (BjornHo) [inline src/main/java/gui/GraphSegment.java]: I think this can be removed.

File: src/main/java/gui/GraphSegment.java [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+//		// TODO Auto-generated constructor stub
+//	}
+//
+//	public GraphSeg(Node... arg0) {
+//		super(arg0);
+//		// TODO Auto-generated constructor stub
+//	}
```
Comment (BjornHo) [inline src/main/java/gui/GraphSegment.java]: I think this can be removed.  You can do that during the merge.

Comment (HugoKooijman) [inline src/main/java/gui/GraphSegment.java]: Actually, the Dnacontent methods can indeed be removed yes. But the getRadius method is used for positioning.
Which gives me an idea, perhaps I can integrate this into the GraphSegment too.

Comment (HugoKooijman) [inline src/main/java/gui/GraphSegment.java]: Nope, the GraphController draws paths relative to the segment coordinates by a translation over the radius. Adding up the radius inside a GraphSegment will not work.

Regardless, I removed the dead code and the unused methods. Should be ready now.

**Comment by HugoKooijman** [comment]
Travis build taking 13 minutes. Details say "Error loading data". Seems to be an issue of Travis itself.

**Comment by HugoKooijman** [comment]
Travis build succes. Thanks for your feedback everyone. Merging PR.

**Pull request merged**